app/src/main/res/menu/login_options_menu.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/loginOptionsEditDelete"
+    android:contentDescription="@string/login_menu_more_options"
+    android:popupMenuStyle="@style/PopupKebabMenu"
+    android:theme="@style/PopupKebabMenu"
+    android:title="@string/login_menu_more_options"
+    app:showAsAction="never" >
+
+    <item
+        android:id="@+id/edit_login_button"
+        android:contentDescription="@string/login_menu_delete_button"
+        android:icon="@drawable/ic_edit"
+        android:minWidth="82dp"
+        android:padding="20dp"
```

Comment:
Same comment as the top of this file, I think customizing the items in the menu are just going to make this menu inconsistent with every other menu in the app